_inc/lib/admin-pages/class.jetpack-my-jetpack-page.php

```diff
@@ -0,0 +1,181 @@
+<?php
+include_once( 'class.jetpack-admin-page.php' );
+include_once( JETPACK__PLUGIN_DIR . 'class.jetpack-modules-list-table.php' );
+
+// Builds the My Jetpack page
+class Jetpack_My_Jetpack_Page extends Jetpack_Admin_Page {
+	// Show the settings page only when Jetpack is connected or in dev mode
+	protected $dont_show_if_not_active = true;
```

Comment:
@zinigor I disagree that the disconnect confirmation should be in a modal. I think we need to start looking at getting away from modals, they provide added complication for mobile and accessibility. :)